p2p/pex_reactor.go

```diff
@@ -240,43 +240,26 @@ func (r *PEXReactor) ensurePeers() {
 		return
 	}
 
-	toDial := make(map[string]*NetAddress)
+	// bias to prefer more vetted peers when we have fewer connections.
+	// not perfect, but somewhate ensures that we prioritize connecting to more-vetted
+	newBias := cmn.MinInt(numOutPeers, 8)*10 + 10
```

Comment:
How come we hoisted this out of the loop? Won't that mean that we'll be using the same bias for every pick? This removes the extra randomness per loop iteration.

Reply:
Unless I'm missing something, it does not appear that `newBias` is influenced by anything that changes with each iteration of the loop - ie. it depends only on `numOutPeers` which is already fixed.

Reply:
Ahaha, my bad. For some reason I looked at that formula and my eyes made me think it was rechoosing the bias. Thank you for the clarification, please disregard my earlier comment.